### PR TITLE
Refactor/readme

### PR DESCRIPTION
**Pull request by uwojeyamudiaga**
# Update Readme

This pull request updates the `README` file to make it look more professional.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,88 @@
...
+The features of the `Develapp` app are -
+- users can find project ideas
+- project ideas are grouped into 
+    * beginner
+    * intermediate
+    * advanced
+- users can be redirected to `chingu` project showcase page
+- users can find project ideas by name
+- users can find project ideas by clicking either one of the three categories on the home page
+- and many more
```
Comment (lgainsbrook) [inline README.md:35]: Do we have more features at this point? I suggest removing this and list all of the current features.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,88 @@
...
+![](https://img.shields.io/badge/Version-v0.0.1-blueviolet?style=plastic&labelColor=inactive)
+![](https://img.shields.io/badge/Stack-MERN-important?style=plastic&labelColor=inactive)
+![](https://img.shields.io/badge/Team-chingu.v38.geckos.04-informational?style=plastic&labelColor=inactive)
+
+<br>
 
 ## Overview
 
-This app was created for the Chingu Voyage 38. 
+One difficulty in learning how to code is figuring out which project to build to improve your skills. Most tutorials simply build a to-do app, probably because the tutors too having a hard time figuring out which projects to build. Perhaps they just want to keep it simple. Develapp is providing solution to developers and tutors who are looking for project ideas to build and include in their portfolio or tutorials. Develapp is made for you so that you can **always be building**.
```
Comment (lgainsbrook) [inline README.md:12]: Suggested edit: "to improve your skills... _Develapp takes you beyond to-do list apps and helps you find project ideas_... to build and include..."

File: README.md (hunk 1 above)
Comment (uwojeyamudiaga) [reply README.md:35]: No, we don't. Just an indication of more to come. But I can remove it, no problem.

File: README.md (hunk 2 above)
Comment (uwojeyamudiaga) [reply README.md:12]: Looks good.

**Comment by uwojeyamudiaga** [comment]
I have made the requested changes. Give it a look and please let me know if any further changes is needed. Thanks

**Review by lgainsbrook — APPROVED** [review]

**Pull request merged**